src/style.css

```diff
@@ -0,0 +1,65 @@
+@font-face {
+  font-family: Avenir-Heavy;
+  src: local("Avenir-Heavy"),
+  url(/src/fonts/Avenir-Heavy.ttf);
+}
+@font-face {
+  font-family: Avenir-Book;
+  src: local("Avenir-Book"),
+  url(/src/fonts/Avenir-Book.ttf);
+}
+
+
+
+body {
+
+  font-weight: 300px;
+  
+}
+
+.search-box {
+
+  display: flex;
+  flex-direction: row;
+  margin-top: 20px;
+  box-shadow: 2px 2px 2px  rgb(176, 176, 176);
+
+}
+
+.smallform {
+  font-family: Avenir-Book;
+  padding: 5px;
+  border: 1px solid rgb(255, 255, 255);
+}
+
+.smallform:focus{
+  outline: 0;
+  border: 1px solid gainsboro;
+}
+
+.bigform{
+  font-family: Avenir-Heavy;
+  padding: 20px;
+  border: 1px solid rgb(255, 255, 255);
+}
```

Comment:
create just one class - .form in you case, and specify additional class .form--small where you need to specify unique attribute values for this class. See checklist https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md